__tests__/helpers/get-all-users.js

```diff
@@ -0,0 +1,31 @@
+const { gql } = require('@apollo/client');
+const client = require('../../utils/apollo-test-client');
+
+module.exports = async (token = null, sort = {}, filter = {}) => {
```

Comment:
Please give this arrow function name and export object with this variable.

Reply:
Done